### PR TITLE
frontend code added

### PR DESCRIPTION
**Pull request by bhavik07**
- default path loads the map
- updated readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,24 +15,32 @@ Contributors include:
 
 To get started with this project:
 
-1. Launch SBT:
+1. Get the [Bower](https://bower.io/) dependencies, to go folder `src/main/resources/webapp`
```
Comment (bhavik07) [inline README.md:18]: If you don't want to install Bower then we can skip this step, by adding all the 2k+ files in the repo once. But it won't be the correct way to do it.

File: src/main/scala/com/flight/FlightService.scala [hunk 2]
```diff
@@ -27,7 +27,21 @@ trait FlightService extends HttpService {
   implicit val formats = DefaultFormats
 
   val route = get {
-    path("isActive") {
+    path("") {
```
Comment (bhavik07) [inline src/main/scala/com/flight/FlightService.scala:32]: i know its polluting the file with many path, but it was the simplest way.

File: README.md (hunk 1 above)
Comment (EXPEjemberton) [reply README.md:18]: I think Bower is fine!

File: src/main/scala/com/flight/FlightService.scala (hunk 2 above)
Comment (EXPEjemberton) [reply src/main/scala/com/flight/FlightService.scala:32]: It's ok, the route is already polluted! :p

**Pull request merged**